src/main/java/org/broadinstitute/hellbender/utils/pileup/ReadPileup.java

```diff
@@ -136,7 +136,7 @@ public ReadPileup getPileupForLane(final String laneID) {
 
     /**
      * Make a new pileup from elements whose reads belongs to a concrete sample
```

Comment:
typo "whose reads belongs to a concrete sample" ->  "whose reads belong to the given sample."